sdks/python/apache_beam/runners/portability/universal_local_runner.py

```diff
@@ -50,42 +37,30 @@
 
 
 class UniversalLocalRunner(runner.PipelineRunner):
-  """A BeamRunner that executes Python pipelines via the Beam Job API.
+  """
+    Experimental: No backward compatibility guaranteed.
+    A BeamRunner that executes Python pipelines via the Beam Job API.
 
-  By default, this runner executes in process but still uses GRPC to communicate
-  pipeline and worker state.  It can also be configured to use inline calls
-  rather than GRPC (for speed) or launch completely separate subprocesses for
-  the runner and worker(s).
+    This runner is a stub and does not run the actual job.
+    This runner schedules the job on a job service. The responsibility of
```

Comment:
Do we need to keep also the old functionality of this class, perhaps as a wrapper?

Reply:
I didn't see a usecase for it other than the test.
We can recreate the functionality when needed.